examples/pixelcnn/train.py

```diff
@@ -144,7 +109,7 @@ def loss_fn(model):
   optimizer = optimizer.apply_gradient(grad, learning_rate=lr)
 
   # Compute exponential moving average (aka Polyak decay)
-  ema_decay = FLAGS.polyak_decay
+  ema_decay = polyak_decay
```

Comment:
Consider using `polyak_decay` directly, or renaming the config attribute to `ema_decay`.

Reply:
Done.